packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -11,12 +11,12 @@
  * Created at     : 2021-01-18 14:53:24 
  * Last modified  : 2021-03-04 21:38:14
  */
-
 import { LitElement, html, property, customElement, internalProperty, query } from 'lit-element';
 import style from './opc-feedback.scss';
 import { repeat } from 'lit-html/directives/repeat.js';
```

Comment:
```suggestion
import { repeat } from 'lit/directives/repeat.js';
```

Reply:
Changed

Reply:
@internalProperty() is changed to @state

<img width="1029" alt="Screenshot 2021-07-13 at 12 44 22 PM" src="https://user-images.githubusercontent.com/13740761/125407993-0f0ec780-e3d8-11eb-9fcb-56dc997b87f1.png">

Reply:
@riginoommen can you change the `@internalProperty` to `@state`